cocos/spine/lib/spine-core.d.ts

```diff
@@ -705,7 +705,7 @@ declare namespace spine {
     }
     class Skin {
         name: string;
-        attachments: Map<Attachment>[];
+        attachments: Map<string, Attachment>[];
```

Comment:
different with https://github.com/EsotericSoftware/spine-runtimes/blob/3.8/spine-ts/build/spine-core.d.ts

Reply:
The definition here is not wrong, but when cocos outputs dts, it does not add the namespace, which causes the map of spine to pollute other code.